tickers_ETH.txt

```diff
@@ -0,0 +1,79 @@
+ZEN
+EOS
+ZEC
```

Comment:
Dont include this in your PR. Most people wont use ETH as base pair

Reply:
it's an absolute pain in the butt to grab custom lists of pairs, maybe leaving this in as an example could really help users.
The amount of the community pairing with ETH is much larger since the USDT value of crypto dipping recently 🙂